modules/client-configuration/pages/clients-alibaba.adoc

```diff
@@ -0,0 +1,125 @@
+[IMPORTANT]
+====
+{alibabaclo} repository URLs are available from {scc}, but the packages and metadata are provided by Alibaba Cloud, not by {suse}.
+====
+endif::[]
+
+
+[IMPORTANT]
+====
+The traditional stack is available on {alibabaclo}{nbsp}2 but it is not supported.
+{alibabaclo}{nbsp}2 clients are only supported as Salt clients.
+====
+
+[WARNING]
+====
+Some {alibabaclo}{nbsp}2 instances will need two tries to register successfully.
+====
+
+== Add Software Channels
+
+Before you register {alibabaclo} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
+
+ifeval::[{suma-content} == true]
+
+The products you need for this procedure are:
+
+[[alibaba-channels-wizard]]
+[cols="1,1", options="header"]
+.Alibaba Cloud Linux Products - WebUI
+|===
+| OS Version | Product Name
+| {alibabaclo} 2 | Alibaba Cloud Linux 2 x86_64
+|===
+
+include::snippets/addchannels_vendor_webui.adoc[]
+
+Alternatively, you can add channels at the command prompt.
+The channels you need for this procedure are:
+
+[[alibaba-channels-suma-cli]]
+[cols="1,1", options="header"]
+.Alibaba Cloud Linux Channels - CLI
+|===
+| OS Version | Base Channel
+| {alibabaclo} 2 | alibaba-2-x86_64
+|===
+
+include::snippets/addchannels_vendor_cli.adoc[]
+
+endif::[]
+
+
+ifeval::[{uyuni-content} == true]
+The channels you need for this procedure are:
+
+[[alibaba-channels-uyuni-cli]]
+[cols="1,1,1", options="header"]
+.Alibaba Cloud Linux Channels - CLI
+|===
+
+| OS Version
+| Core Channel
+| Client Channel
+
+| {amazon} 2
+| alibaba-2
+| alibaba-2-uyuni-client
+
+|===
+
+
+include::snippets/addchannels_novendor_cli.adoc[]
+
+[NOTE]
+====
+The client tools channel provided by [command]``spacewalk-common-channels`` is sourced from {uyuni} and not from {suse}.
+====
+
+endif::[]
+
+== Check Synchronization Status
+
+ifeval::[{suma-content} == true]
+
+include::snippets/check_sync_webui_suma.adoc[]
+
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+
+include::snippets/check_sync_webui_uyuni.adoc[]
+
+endif::[]
+
+include::snippets/check_sync_cli.adoc[]
+
+
+
+== Create an Activation Key
+
+You need to create an activation key that is associated with your {alibabaclo} channels.
+
+For more information on activation keys, see xref:client-configuration:activation-keys.adoc[].
+
+
+
+== Register Clients
+
+{alibabaclo} clients are registered in the same way as all other clients.
+
+Some {alibabaclo}{nbsp}2 instances will fail to register on the first try due
+to a bug in the {alibabaclo}{nbsp}2 image, where `python-urlgrabber3` is 
+provided both as a pip (native Python) package and regular RPM package. This 
+causes a conflict at the filesystem level and an error when you first try to 
+register. If your instance is based on one of the affected image versions, you 
+should try to register the client for a second time before reporting a bug.
+
+For more information, see xref:client-configuration:registration-overview.adoc[].
```

Comment:
```suggestion
For more information about client registration, see xref:client-configuration:registration-overview.adoc[].
```

Only because we've broken the flow with the admon.